_modules/dotnet-kill.js

```diff
@@ -0,0 +1,67 @@
+// -----------------------------------------------------------------------------------------
+// #region Imports
+// -----------------------------------------------------------------------------------------
+
+const child_process = require("child_process");
+const echo          = require("./echo");
+const ps            = require("./ps");
+const shell         = require("shelljs");
+
+// #endregion Imports
+
+// -----------------------------------------------------------------------------------------
+// #region Functions
+// -----------------------------------------------------------------------------------------
+
+const dotnetKill = {
+    cmd() {
+        return {
```

Comment:
Not suggesting we change this now, but if this "pattern" results in adding more properties/methods than those here, I feel we need to extract a 'command/cmd' object type/class

Reply:
Not a bad idea. I like it a lot more than doing the string building/splitting manually, but I could see this manifesting itself in some sort of builder or type.